src/integrations/fb_signer.ts

```diff
@@ -101,52 +106,54 @@ export class FbSigner {
   }
 
   /**
-   * Sign a generic transaction with fireblocks using Fireblocks raw message signing feature.
-   * @param payloadToSign: transaction data in hexadecimal
-   * @param derivationPath: derivation path of the token to sign
-   * @param algorithm: algorithm of the token to sign
-   * @param note: optional fireblocks custom note
+   * Sign an EIP-712 Ethereum typed message with fireblocks
+   * @param eip712message eip712message to sign
+   * @param assetId fireblocks asset id
+   * @param note optional fireblocks custom note
    */
-  public async signGenericWithFB(
-    payloadContent: string,
-    derivationPath: number[],
-    algorithm: SigningAlgorithm,
+  public async signTypedMessage(
```

Comment:
We still need the derivation path option for COSMOS based protocols.

Reply:
in sign() you can pass it in the payloadToSign argument

Reply:
github diff make things weird but I removed signGenericWithFB which can be done with sign.
signTypedMessage is for signing ethereum message (not even used yet)